### PR TITLE
add transfer document-store transferOwnership command

### PR DESCRIPTION
**Pull request by yapyuyou**
Story - https://www.pivotaltracker.com/n/projects/2464296/stories/178684134

### CODE REVIEW

**Comment by Nebulis** [comment]
Can't see th story =)

File: src/commands/document-store/transferOwnership.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { Argv } from "yargs";
```
Comment (Nebulis) [inline src/commands/document-store/transferOwnership.ts:1]: kebab-case for the filename =)

File: src/commands/document-store/transferOwnership.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { error, info, success } from "signale";
+import { getLogger } from "../../logger";
+import { transferOwnershipToDocumentStore } from "../../implementations/document-store/transferOwnership";
+import { DocumentStoreTransferOwnershipCommand } from "./document-store-command.type";
+import { withGasPriceOption, withNetworkAndWalletSignerOption } from "../shared";
+import { getEtherscanAddress } from "../../utils";
+
+const { trace } = getLogger("document-store:transferOwnership");
+
+export const command = "transferOwnership [options]";
```
Comment (Nebulis) [inline src/commands/document-store/transferOwnership.ts:11]: transfer-ownership same kebab-case

File: src/commands/document-store/transferOwnership.ts (hunk 1 above)
Comment (yapyuyou) [reply src/commands/document-store/transferOwnership.ts]: oops my bad! Changed

File: src/commands/document-store/transferOwnership.ts (hunk 2 above)
Comment (yapyuyou) [reply src/commands/document-store/transferOwnership.ts]: changed

File: src/implementations/document-store/transfer-ownership.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import signale from "signale";
+import { getLogger } from "../../logger";
+import { DocumentStoreTransferOwnershipCommand } from "../../commands/document-store/document-store-command.type";
+import { getWalletOrSigner } from "../utils/wallet";
+import { dryRunMode } from "../utils/dryRun";
+import { TransactionReceipt } from "@ethersproject/providers";
+
+const { trace } = getLogger("document-store:transfer-ownership");
+
+export const transferOwnershipToDocumentStore = async ({
```
Comment (ChenRen93) [inline src/implementations/document-store/transfer-ownership.ts:11]: hi, the naming seems to be different from the functionality and description. it is more towards transferDocumentStoreOwnershipToWallet ?

Comment (yapyuyou) [reply src/implementations/document-store/transfer-ownership.ts]: hello! agreed and changed

**Review by Nebulis — APPROVED** [review]

**Comment by Nebulis** [comment]
@yapyuyou be careful when you squash, make sure your commit message won't break commitlint =)

**Pull request merged**